conftest.py

```diff
@@ -244,7 +260,9 @@ def get_test_launcher() -> str:
 
     @staticmethod
     def get_test_port() -> int:
-        return test_port
+        # TODO: Ideally this should find a free port on the correct host(s),
```

Comment:
Is a ticket created for this? If not, possibly might forget!

Reply:
[Added](https://github.com/CrayLabs/Poseidon/issues/691)!!